lib/urlwatch/storage.py

```diff
@@ -520,8 +520,8 @@ def load(self, job, guid):
 
         return data, timestamp, None, None
 
-    def save(self, job, guid, data, timestamp, etag=None):
-        # Timestamp and ETag are always ignored
+    def save(self, job, guid, data, timestamp, tries, etag=None):
+        # Timestamp, tries and ETag are always ignored
```

Comment:
Is this an unrelated change that fixes another error?

Reply:
Yes, this was missing in the file cache storage as its deprecated, but I was using it to test if minidb was the cause of the threading issues.